Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionTabCocoa.mm

```diff
@@ -180,12 +198,28 @@
     return [m_delegate isPinnedForWebExtensionContext:m_extensionContext->wrapper()];
 }
 
-void WebExtensionTab::toggleReaderMode()
+bool WebExtensionTab::isPrivate() const
+{
+    if (!isValid())
```

Comment:
Do we want a `m_respondsToIsPrivate` and to check that here?

Reply:
Not needed, since we ask the window and it does the check. Though I think we should ask the window if it responds to avoid some work. Will do that.